pkg/pki/minisign/minisign_test.go

```diff
@@ -44,14 +44,33 @@ func TestReadPublicKey(t *testing.T) {
 	}
 
 	for _, tc := range tests {
-		file, err := os.Open(tc.inputFile)
-		if err != nil {
-			t.Errorf("%v: cannot open %v", tc.caseDesc, tc.inputFile)
-		}
+		t.Run(tc.caseDesc, func(t *testing.T) {
```

Comment:
It might make sense to add more test cases in the tests slice above

Reply:
Or do we cover all added new functionality?

Reply:
Reorganized the tests a bit and added one more case!